compiler/passes/src/type_checking/program.rs

```diff
@@ -241,15 +241,88 @@ impl ProgramVisitor for TypeCheckingVisitor<'_> {
     }
 
     fn visit_function(&mut self, function: &Function) {
+        // Set type checker variables for function variant details.
+        self.scope_state.initialize_function_state(function.variant);
+
         // Check that the function's annotations are valid.
-        // Note that Leo does not natively support any specific annotations.
+
         for annotation in function.annotations.iter() {
-            // TODO: Change to compiler warning.
-            self.emit_err(TypeCheckerError::unknown_annotation(annotation, annotation.span))
+            if !matches!(annotation.identifier.name, sym::native_test | sym::interpreted_test | sym::should_fail) {
```

Comment:
Because we now have `interpret` procedures, coul we get rid of `native_test` and `interpreted_test` and use `test`?